tests/test_tools_export_jsonl.py

```
@@ -0,0 +1,112 @@
+
+
+def test_export_jsonl_invalid_file(tmp_path: Path) -> None:
+    output_file = tmp_path / "output.jsonl"
+
+    non_existent = tmp_path / "not_exist.json"
+    with pytest.raises(FileNotFoundError, match="File not found"):
+        export_jsonl(non_existent, output_file)
+
+    not_json = tmp_path / "input.txt"
+    not_json.write_text("plain text", encoding="utf-8")
+    with pytest.raises(ValueError, match="must be a .json file"):
```

Ruff (0.16.2)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)